rising/utils/affine.py

```diff
@@ -106,28 +104,6 @@ def points_to_cartesian(batch: torch.Tensor) -> torch.Tensor:
     return batch[..., :-1] / batch[..., -1, None]
 
 
-def matrix_revert_coordinate_order(batch: torch.Tensor) -> torch.Tensor:
```

Comment:
we will need this, when we revert the order by a flag (as we discussed) to optionally make it consistent with PyTorch